config/openshift/kustomization.yaml

```diff
@@ -0,0 +1,43 @@
+# Adds namespace to all resources.
+namespace: openshift-kubevirt-ipam-controller-system
```

Comment:
@oshoval / @qinqon should this be `openshift-cnv` ??

I don't think so, since this method of installation is just for testing this works without having to grab the whole CNV bundle (i.e. smoke tests, assert this deploys, etc).

Reply:
i think you can just do whatever ns yo uwant
if it helps, this worked for me i think
https://gist.github.com/oshoval/d5db815ebea60eb1777a838bee0530b9#file-gistfile1-txt-L14

might just need to add the scc fix for 4.19